client/src/styles/Footer.css

```diff
@@ -0,0 +1,28 @@
+.footer {
+  display: flex;
+  position: fixed;
+  left: 0;
+  bottom: 0;
+  width: 100%;
+  background-color: black;
+  color: white;
+  text-align: center;
+  padding: 10px;
+}
+
+img {
```

Comment:
```suggestion
.footer-image {
```
It's not considered good practise to use HTML elements as the CSS selector. Instead, we should use classes as the CSS selector — let's update this